java-api-core/src/com/kinvey/java/network/NetworkManager.java

```diff
@@ -950,10 +960,18 @@ public T execute() throws IOException {
      */
     public class GetCount extends AbstractKinveyReadHeaderRequest<KinveyCountResponse> {
 
-        private static final String REST_PATH = "appdata/{appKey}/{collectionName}/_count";
```

Comment:
Are we removing this `REST_PATH`? Do we still need it for the `GetCount()` where no query is passed?

Reply:
New `REST_PATH` works either with query and without it.